pytorch/buildspec-graviton.yml

```diff
@@ -31,9 +34,22 @@ images:
     build: &PYTORCH_CPU_INFERENCE_PY3 false
     image_size_baseline: 10000
     device_type: &DEVICE_TYPE cpu
+    os_version: &OS_VERSION ubuntu20.04
     python_version: &DOCKER_PYTHON_VERSION py3
     tag_python_version: &TAG_PYTHON_VERSION py38
     tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION , "-ec2"]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton.ec2., *DEVICE_TYPE ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton., *DEVICE_TYPE ]
     context:
       <<: *INFERENCE_CONTEXT
```

Comment:
Is this spec missing the `target` input, to tell it that the build target is only `ec2`?

Reply:
added